packages/dynamite/dynamite/lib/src/models/type_result/enum.dart

```diff
@@ -20,7 +20,7 @@ class TypeResultEnum extends TypeResult {
     final bool onlyChildren = false,
     final String? mimeType,
   }) {
-    if (subType.name == 'String') {
+    if (subType.name == 'String' || subType.name == 'int') {
       return '$object.name';
     }
```

Comment:
Shouldn't this return the name in all cases regardless of the subtype? I guess that was left over from the changes I did

Reply:
I'd rather merge #1323 as it will remove this mess altogether :)

Reply:
Ok let's get that merged first then